packages/runtime-dom/src/nodeOps.ts

```diff
@@ -13,7 +14,23 @@ export const nodeOps: Omit<RendererOptions<Node, Element>, 'patchProp'> = {
 
   remove: child => {
     const parent = child.parentNode
+    function removeVEI (node: any) {
```

Comment:
I'm not sure it's a good place to remove events, maybe another place is better, but basic idea is like this.